src/server/plugins/engine/helpers.ts

```diff
@@ -77,3 +79,7 @@ export const filesize = (bytes: number) => {
 
   return Math.max(bytes, 0.1).toFixed(1) + byteUnits[i]
 }
+
+export const checkIfPreview = (path: string) => {
```

Comment:
Mind if we use a named function for stack traces and add some tests?

```suggestion
export function hasPreviewPath(path: string) {
```